imports/lib/schemas/DocumentActivity.ts

```diff
@@ -7,10 +7,11 @@ import { Overrides, buildSchema } from './typedSchemas';
    server, not by users */
 export const DocumentActivityCodec = t.type({
   _id: t.string,
-  ts: date, /* 5 minute granularity */
+  ts: date, /* rounded to ACTIVITY_GRANULARITY */
   hunt: t.string,
   puzzle: t.string,
   document: t.string,
+  user: t.union([t.undefined, t.string]),
```

Comment:
Maybe comment on this field is expected to be `undefined` if we don't recognize the google user ID, which can happen in normal operation.